hvplot/converter.py

```diff
@@ -842,13 +842,22 @@ def __init__(
         if kind == 'errorbars':
             hover = False
         elif hover is None:
-            hover = not self.datashade
+            hover = True
```

Comment:
```suggestion
            hover = (self.datashade and self.kind == 'points') or not self.datashade
```
I'm not sure if hover should always be True.